bin/common/create_kubedns.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echo "Waiting for kubectl to respond..."
+while ! kubectl get pods --all-namespaces; do
+  sleep 2
+done
+echo "Waiting for kube-apiserver to be active..."
+while ! systemctl is-active kube-apiserver.service 2>/dev/null >/dev/null; do
```

Comment:
Fixed the order of these: https://github.com/SUSE/scf/blob/harts-vagrant-use-public-network-rebased/packer/scripts/create-kubedns.sh#L5-L12